src/execution/execute.js

```diff
@@ -594,7 +622,12 @@ function resolveOrError(
   } catch (error) {
     // Sometimes a non-error is thrown, wrap it as an Error for a
     // consistent interface.
-    return error instanceof Error ? error : new Error(error);
+    const wrappedError = error instanceof Error ? error : new Error(error);
+    const pathedError = new PathedError(
```

Comment:
I'm not sure if this is the correct place for this information to be added. It should probably instead be added where GraphQLError is constructed.